app/views/membership.html

```diff
@@ -207,27 +235,7 @@
                       <div
                         class="col-roles"
                         row tablet="column" flex wrap axis="start">
-                        <div
-                          ng-if="newBinding.kind === 'ServiceAccount'"
-                          class="service-account-namespace hidden-xs">
-                          <ui-select
-                            ng-model="newBinding.namespace"
-                            theme="bootstrap"
-                            search-enabled="true"
-                            title="Select a project"
-                            class="select-project">
-                            <ui-select-match
-                              placeholder="Select a project">
-                              <span ng-bind="newBinding.namespace"></span>
-                            </ui-select-match>
-                            <ui-select-choices
-                              repeat="projectName in projects | filter: $select.search"
-                              refresh="refreshProjects($select.search)"
-                              refresh-delay="200">
-                              <div>{{projectName}}</div>
-                            </ui-select-choices>
-                          </ui-select>
-                        </div>
+                        &nbsp;
```

Comment:
Yeah, to keep the flex from collapsing completely, habit, I think its irrelevant, I'll remove & the extra axis stuff.